docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

```diff
@@ -205,8 +206,8 @@ or selectively configure single elements.
 
 ## kubeadm alpha phase upload-config {#cmd-phase-upload-config}
 
-You can use this command to upload the configuration of your cluster. Alternatively, you 
-can use [kubeadm config](kubeadm-config.md).
+You can use this command to upload the kubeadm-specific configuration/desired state of your cluster.
```

Comment:
"You can use this command to upload the kubeadm configuration to your cluster."

Reply:
@luxas ☝️

Reply:
fixed